app/payment-request/get-payment-request.js

```diff
@@ -23,15 +24,17 @@ const getPaymentRequest = async (categoryId = 1) => {
       'value',
       'received',
       'receivedFormatted',
-      'ledger'
+      'ledger',
+      'netValue'
     ]
   })
 }
 
 const getPaymentRequestByInvoiceNumber = async (invoiceNumber) => {
   return db.paymentRequest.findOne({
     where: {
-      invoiceNumber
+      invoiceNumber,
+      ledger: AR
```

Comment:
This changes the functionality and expected functionality of this function. Would recommend reverting this named function back and creating a new named function `getARPaymentRequestByInvoiceNumber` which also filters for those with `ledger: AR`